neqo-transport/src/streams.rs

```diff
@@ -488,25 +498,24 @@ impl Streams {
     }
 
     pub fn set_initial_limits(&mut self) {
-        _ = self.local_stream_limits[StreamType::BiDi].update(
-            self.tps
-                .borrow()
-                .remote()
-                .get_integer(InitialMaxStreamsBidi),
-        );
+        let remote = self.tps.borrow();
+        let remote = remote.remote();
```

Comment:
The name shadowing is unfortunate in this case, because the first value needs to remain in scope, it's not totally shadowed.